README.md

```diff
@@ -16,9 +12,16 @@ Each example has an README.md that explains what each project does and how to wo
   - [Kedro 0.15.8](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.15.8)
   - [Kedro 0.16.1](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.1)
   - [Kedro 0.16.4](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.4)
-- Documentation that you can use to deliver Kedro training in [`kedro-training`](https://github.com/quantumblacklabs/kedro-examples/tree/master/kedro-training)
 
-## Where can I find more `kedro-examples`?
+
+## Where can I find more learning material and `kedro-examples`?
+
+The [`Kedro-Training`](https://github.com/quantumblacklabs/kedro-training) repository contains documentation that you can use to deliver Kedro training.
+
+These repositories contain the official Kedro tutorial projects, also known as starters:
+- [Space Flights tutorial](https://github.com/quantumblacklabs/kedro-starter-spaceflights)
```

Comment:
```suggestion
- [Spaceflights tutorial](https://github.com/quantumblacklabs/kedro-starter-spaceflights)
```

Capitalised here because it's at the start of a bullet :)